app/controllers/api/v1/users.rb

```diff
@@ -0,0 +1,39 @@
+module API
+  module V1
+    class Users < Grape::API
+      include API::V1::Defaults
+      resource :users do
```

Comment:
- [ ] I should be able to create a user or login to a user account. (Login endpoint, Registration endpoint)